front/src/components/boxs/chart/Chart.jsx

```diff
@@ -133,40 +137,82 @@ class Chartbox extends Component {
     }
     await this.setState({ loading: true });
     try {
+      const maxStates = 300;
```

Comment:
So in the end maxStates is 300 for everything?

Reply:
Yes, 300 is the max in binary where everything goes well in display (no lag). As I had put it for the other graphs we could have put 3000, but as it is not the object of the PR and that it works very well for all, I preferred to put a single value for all the graphs for the moment, we will be able to go up in a separate PR with a greater control of tests on the other type of graph. Knowing that I am working on the expand ... to see?